packages/amplify-graphql-api-construct/src/sql-model-datasource-strategy.ts

```diff
@@ -0,0 +1,64 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { SQLLambdaModelDataSourceStrategy, SqlModelDataSourceDbConnectionConfig } from './types';
+
+/**
+ * Type predicate that returns true if the object is a SQLLambdaModelDataSourceStrategy.
+ * @param obj the object to inspect
+ * @returns true if the object is shaped like a SQLLambdaModelDataSourceStrategy
+ */
+export const isSQLLambdaModelDataSourceStrategy = (obj: any): obj is SQLLambdaModelDataSourceStrategy => {
+  console.log(`obj: ${JSON.stringify(obj)}`);
```

Comment:
Is this additional logging intentional?

Reply:
Whoops, that was in the previous definition, an artifact of troubleshooting. Will remove.